### ISSUE TITLE
We should work on this

### ISSUE THREAD

**Issue by nathanhilbert** [brandomr/speechscrape#1]
@brandomr 
This has been on my back burner for a long time.  Let's see what we can come up with.

**Comment by nathanhilbert** [comment]
Forgot... 
https://github.com/USStateDept/speechscrape/tree/master/CLAVIN

**Comment by brandomr** [comment]
Absolutely! I'll check out the CLAVIN work and move over to the State repo....would be very cool to visualize location mentions over time. Was also thinking of using LDA to model topics discussed over time.

**Comment by nathanhilbert** [comment]
I requested that you are given write access to the github DoS org so you can push directly.
Can we do a hack day on this?  I'm available this Friday or next Friday most of the day.  Or if you prefer, we can continue asynchronously.

**Comment by nathanhilbert** [comment]
Apparently, you are a member of the DoS org, but the badge is not on your profile and I can't see you in the members list (only admins can see you).  Please verify that you can in fact push or start a repo on the DoS org.

**Comment by nathanhilbert** [comment]
FYI came along this.  I haven't looked at it.
https://github.com/pudo/aleph

**Comment by brandomr** [comment]
let's definitely try to get together next Friday. I'll keep you posted how the week goes as I'm out this Friday and next Monday. 

I wasn't able to push to the speechscrape repo, but was able to create a new repo (I deleted it) under USStateDept which is strange...
